packages/fast-components-styles-msft/src/toggle/index.ts

```diff
@@ -1,21 +1,38 @@
-import { IDesignSystem } from "../design-system";
+import designSystemDefaults, { IDesignSystem } from "../design-system";
 import { ComponentStyles, ComponentStyleSheet, ICSSRules } from "@microsoft/fast-jss-manager";
 import { applyLocalizedProperty, toPx } from "@microsoft/fast-jss-utilities";
 import { typeRamp } from "../utilities/typography";
 import { IToggleClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { get } from "lodash";
 import * as Chroma from "chroma-js";
 import { Direction } from "@microsoft/fast-application-utilities";
 
 function setFloatAndClear(clearFloat: boolean = true): ICSSRules<IDesignSystem> {

```

Comment:
I think you should separate float and clear. It isn't super intuitive from a readers perspective as to what this does when any particular param is passed.

If you make two functions (each one does one thing) and uses them when needed you will increase readability and reduce code complexity.

Also, since these are localizing floats and clears, we should probably make that clear with the function name.

Reply:
Updated based on this feedback.